indexing-service/src/main/java/io/druid/indexing/common/task/IndexTask.java

```diff
@@ -898,6 +899,7 @@ public boolean isAppendToExisting()
     private static final boolean DEFAULT_GUARANTEE_ROLLUP = false;
     private static final boolean DEFAULT_REPORT_PARSE_EXCEPTIONS = false;
     private static final long DEFAULT_PUBLISH_TIMEOUT = 0;
+    private static final Period DEFAULT_LOCK_TIMEOUT = new Period("PT5m");
```

Comment:
Yes, it was intended because tasks of different types use different tuningConfig implementation. But now, I'm thinking to move the ```lockTimeout``` configuration to ```TaskContext``` (https://github.com/druid-io/druid/pull/1604). It would be better because this configuration is used by all types of tasks, and thus moving to ```TaskContext``` will reduce duplicated codes in each implementation.